cmd/charm/charmcmd/push.go

```diff
@@ -437,6 +437,7 @@ func parseBzrLogEntry(entryText string) (bzrLogEntry, error) {
 		val := strings.TrimLeft(kvp[1], " ")
 		switch kvp[0] {
 		case "revno":
+			val = strings.TrimSuffix(val, " [merge]")
```

Comment:
This works. Could also just remove the space and anything else following the revno digits (`s/ .*//` for example), not knowing or caring much about the possible things bzr could append to the revno.